web/beacon-app/src/features/projects/components/EditProject/__tests__/EditProjectForm.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+
+import type { Project } from '@/features/projects/types/Project';
+
+import EditProjectForm from '../EditProjectForm';
+// import selectEvent from 'react-select-event';
+
+const renderComponent = (props) => {
+  const queryClient = new QueryClient();
+  const wrapper = ({ children }) => (
+    <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+  );
+  return render(<EditProjectForm {...props} />, { wrapper });
+};
+
+vi.mock('@lingui/macro', () => ({
+  t: (str) => str,
+  Trans: ({ children }) => children,
+}));
+// vi.mock('@/features/members/hooks/useMembers', () => ({
```

Comment:
I see that this code has been commented out. Will this test need this in the future?